controllers/student.js

```diff
@@ -0,0 +1,41 @@
+const bcrypt = require('bcryptjs');
+const BigPromise = require('../middlewares/BigPromise.js');
+const User = require('../models/userSchema.js');
+
+const registerStudent = BigPromise(async (req, res, next) => {
+  // TODO : What to do with Contact?
+  let { name, email, department, role, id, password, confirmPassword } = req.body;
+
+  if (!name || !email || !department || !role || !id || !password || !confirmPassword) {
```

Comment:
We can keep this validation on frontend. We will only allow user to submit the form once he/she fills all these information